### PR TITLE
create student section

### PR DESCRIPTION
**Pull request by ShivamRai2003**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: Student Section [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- name: Shivam Rai
+  country:India
+  github: ShivamRai2003
+  image: Picture 090.jpg
+  twitter: Shivam Rai
```
Comment (nikit19) [inline Student Section]: Just add these lines in the students.yml file in the _data folder.
You don't have to write the rest

File: Student Section [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+@ShivamR44888706
+
+**Optional** (Please ensure parents agree on sharing this data)
```
Comment (codedsun) [inline Student Section]: Either fill it or remove it .

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please upload the photo and make these changes in the students.yml file

**Comment by ShivamRai2003** [comment]
how to do please tell me

File: Student Section (hunk 1 above)
Comment (ShivamRai2003) [reply Student Section]: how to upload a photo there is no option

**Comment by nikit19** [comment]
You need to upload the photo in your forked repository

**Comment by ShivamRai2003** [comment]
can you send the screenshot please

**Comment by ShivamRai2003** [comment]
i cannot add my photo only please tell what to do

File: Student Section (hunk 1 above)
Comment (akarshan96) [reply Student Section]: @ShivamRai2003 just add the image to img/students/ directory of the forked repository.

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Dont make multiple PRs for the same issue

**Pull request closed without merging**